bigbluebutton-html5/imports/api/breakouts/server/modifiers/clearBreakouts.js

```diff
@@ -0,0 +1,24 @@
+import Breakouts from '/imports/api/breakouts';
+import Logger from '/imports/startup/server/logger';
+import removeMeeting from '/imports/api/meetings/server/modifiers/removeMeeting';
+
+export default function clearBreakouts(meetingId) {
+  let cb = () => {
```

Comment:
Consider moving this callback for remove all to the end of the function (or in an `else`)